package.json

```diff
@@ -6,8 +6,6 @@
   "scripts": {
     "prepublish": "npm run build",
     "build": "rm -rf lib && babel -d lib src",
-    "test": "mocha --require test/_setup --compilers js:babel-core/register --recursive",
```

Comment:
`npm test` should continue to work - can you set this to `npm run --silent karma`?